pkgs/by-name/gi/github-distributed-owners/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "github-distributed-owners";
+  version = "0.1.11";
+
+  src = fetchFromGitHub {
+    owner = "andrewring";
+    repo = "github-distributed-owners";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-oLRcH1lgRxlYIlyk3bPWO5YmCIq462YUjBjMSPOF7Ic=";
+  };
+
+  cargoHash = "sha256-pt/GoXF/uSU78pZqG8PgFe+tlbwZH2qpGQD7jgC52NM=";
+
```

can be tested with `nix run nixpkgs#nix-update -- github-distributed-owners`, will make a pr you can merge when a new release is published
```suggestion

  passthru.updateScript = nix-update-script { };

```

---

Thanks! Ran it locally and appear to work.

```bash
 nix run nixpkgs#nix-update -- github-distributed-owners
$ nix eval --json --impure --expr '
let
  
  pkgs = import "/Users/cameronmora/workspace/nixpkgs";
  args =  builtins.functionArgs pkgs;
  inputs = (if args ? system then { system = builtins.currentSystem; } else {}) //
           (if args ? overlays then { overlays = [ ]; } else {});
  pkg = (pkgs inputs)."github-distributed-owners";
  sanitizePosition = x: x;

  positionFromMeta = pkg: let
    parts = builtins.match "(.*):([0-9]+)" pkg.meta.position;
  in {
    file = builtins.elemAt parts 0;
    line = builtins.fromJSON (builtins.elemAt parts 1);
  };

  raw_version_position = sanitizePosition (builtins.unsafeGetAttrPos "version" pkg);

  position = if pkg ? isRubyGem then
    raw_version_position
  else if pkg ? isPhpExtension then
    raw_version_position
  else if (builtins.unsafeGetAttrPos "src" pkg) != null then
    sanitizePosition (builtins.unsafeGetAttrPos "src" pkg)
  else
    sanitizePosition (positionFromMeta pkg);
in {
  name = pkg.name;
  old_version = pkg.version or (builtins.parseDrvName pkg.name).version;
  inherit raw_version_position;
  filename = position.file;
  line = position.line;
  urls = pkg.src.urls or null;
  url = pkg.src.url or null;
  rev = pkg.src.rev or null;
  tag = pkg.src.tag or null;
  hash = pkg.src.outputHash or null;
  go_modules = pkg.goModules.outputHash or null;
  go_modules_old = pkg.go-modules.outputHash or null;
  cargo_deps = pkg.cargoDeps.outputHash or null;
  cargo_vendor_deps = pkg.cargoDeps.vendorStaging.outputHash or null;
  raw_cargo_lock =
    if pkg ? cargoDeps.lockFile then
      let
        inherit (pkg.cargoDeps) lockFile;
        res = builtins.tryEval (sanitizePosition {
          file = toString lockFile;
        });
      in
      if res.success then res.value.file else false
    else
      null;
  composer_deps = pkg.composerVendor.outputHash or null;
  composer_deps_old = pkg.composerRepository.outputHash or null;
  npm_deps = pkg.npmDeps.outputHash or null;
  pnpm_deps = pkg.pnpmDeps.outputHash or null;
  yarn_deps = pkg.yarnOfflineCache.outputHash or null;
  yarn_deps_old = pkg.offlineCache.outputHash or null;
  maven_deps = pkg.fetchedMavenDeps.outputHash or null;
  has_nuget_deps = pkg ? nugetDeps;
  mix_deps = pkg.mixFodDeps.outputHash or null;
  tests = builtins.attrNames (pkg.passthru.tests or {});
  has_update_script = pkg.passthru.updateScript or null != null;
  src_homepage = pkg.src.meta.homepage or null;
  changelog = pkg.meta.changelog or null;
  maintainers = pkg.meta.maintainers or null;
}' --extra-experimental-features 'flakes nix-command'
warning: Nix search path entry '/nix/var/nix/profiles/per-user/root/channels' does not exist, ignoring
fetch https://github.com/andrewring/github-distributed-owners/releases.atom
Not updating version, already 0.1.11
Package maintainers:
  - Cameron Byte (@cameroncuttingedge)
$ git -C /Users/cameronmora/workspace/nixpkgs diff -- /Users/cameronmora/workspace/nixpkgs/pkgs/by-name/gi/github-distributed-owners/package.nix
No changes detected, skipping remaining steps
```